kerchunk/nos-sqs-sensor.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Sensor
+metadata:
+  name: nos-sqs
+spec:
+  template:
+    serviceAccountName: operate-workflow-sa
+  dependencies:
+    - name: nos-dep
+      eventSourceName: nos-sqs
+      eventName: newofs
+  triggers:
+    - template:
+        name: nos-sqs-workflow
+        k8s:
+          operation: create
+          source:
+            resource:
+              apiVersion: argoproj.io/v1alpha1
+              kind: Workflow
+              metadata:
+                generateName: nos-sqs-workflow-
+              spec:
+                entrypoint: extract-sqs-message
+                arguments:
+                  parameters:
+                  - name: message
+                    # this is the value that should be overridden
+                    value: hello world
+                templates:
+                - name: extract-sqs-message
+                  inputs:
+                    parameters:
+                    - name: message
+                  script:
+                    image: miannuccirps/kerchunk-nos:latest
```

Comment:
Could you push this image to a new public ECR in AWS? I've been using the annotation nextgen-dmac/<image name>

Reply:
Yeah I'll play with that next